code/modules/research/designs/weapon_designs.dm

```diff
@@ -126,7 +126,7 @@
 	category = list("Weapons")
 
 /datum/design/smg
-	name = "Prototype Submachine Gun"
+	name = "NanoTrasen Saber SMG"
```

Comment:
What duncathan said. It appears in the protolathe menu, but since we can't actually examine names in a menu, the \improper doesn't matter. Everything else in the protolathe/circuit imprinter doesn't use it either.